automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/GroupFixtures.scala

```diff
@@ -12,6 +12,8 @@ import org.scalatest.Suite
   */
 trait GroupFixtures extends CleanUp with LazyLogging { self: WebBrowserSpec with Suite =>
 
+  def groupNameToEmail(groupName: String): String = s"GROUP_$groupName@quality.firecloud.org" //probably want to pull the apps domain from config if it's there
```

Comment:
just realized i hadn't resolved this comment yet, i'll need to add this to `application.conf`